--- ./Makefile.org	Sat Oct 27 11:34:00 2001
+++ ./Makefile	Thu Nov  1 15:52:19 2001
@@ -1,15 +1,21 @@
-CFLAGS= -fpic -Wall `gtk-config --cflags` `imlib-config --cflags-gdk`
+CFLAGS= -fpic -Wall `gtk-config --cflags` `imlib-config --cflags-gdk` -I/usr/X11R6/include
 LDFLAGS= -shared -Wl
 
 # uncomment the second DBGFLAGS line to make seti@home plugin
 # be verbose when GKrellM is launched with the --debug option.
 DBGFLAGS= -O3
 DBGFLAGS= -DDEBUG -Wall
+PREFIX=/usr/X11R6
 
 all: seti.so
 
 install: seti.so
-	cp seti.so ${HOME}/.gkrellm/plugins/
+	install -d ${DESTDIR}${PREFIX}/lib/gkrellm
+	install seti.so ${DESTDIR}${PREFIX}/lib/gkrellm
+
+install-home: seti.so
+	install -d ${HOME}/.gkrellm/plugins/
+	install seti.so ${HOME}/.gkrellm/plugins/
 
 seti.so: seti.o
 	gcc $(LDFLAGS) -o seti.so seti.o
--- ./INSTALL.org	Thu Nov  1 15:54:06 2001
+++ ./INSTALL	Thu Nov  1 15:55:08 2001
@@ -25,11 +25,14 @@
 
 Then, you should just have to type
 
-	make install
+	make install-home
 
 This will compile the plugin and copy it under ~/.gkrellm/plugins/
 directory.
 
+If you want to have this plugin avaible for all people in your system, type:
+	make install
+
 If you want to do it by hand, because you encounter problems or
 whatever, process through the following steps:
 
